appservice/src/main/java/nu/yona/server/subscriptions/rest/BuddyController.java

```diff
@@ -291,8 +291,11 @@ public BuddyResource toResource(BuddyDTO buddy)
 			ControllerLinkBuilder selfLinkBuilder = getSelfLinkBuilder(buddy.getID());
 			addSelfLink(selfLinkBuilder, buddyResource);
 			addEditLink(selfLinkBuilder, buddyResource);
-			addDayActivityOverviewsLink(buddyResource);
-			addWeekActivityOverviewsLink(buddyResource);
+			if (buddy.getSendingStatus() == Status.ACCEPTED)
```

Comment:
Could you add a javadoc comment to explain this is the buddy who sends (and not the user having the buddy)? In the Swagger spec too (two places).